events.html

```diff
@@ -0,0 +1,131 @@
+--- layout: default ---
+<div class="hero">
+    <div class="hero-body">
+        <h1 class="title">
+            Porto Event Listing
+        </h1>
+    </div>
+</div>
+<div class="content">
+    <p>
+        We believe in building bridges all tech related groups in the region, in order to create more momentum in the tech scene
```

Comment:
We believe in building bridges **between** all tech...

Reply:
Thanks. There's always a typo somewhere. It's a trademark of mine :)